### PR TITLE
feat: added reporting to e2e tests

### PR DESCRIPTION
**Pull request by nekhvoya**

### CODE REVIEW

File: test/e2e/pinnwand.toml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-database_uri = "sqlite:///C:\\Users\\peeka\\AppData\\Local\\Temp\\tmpdxas7neb"
+database_uri = "sqlite:///C:\\Users\\peeka\\AppData\\Local\\Temp\\tmpbxb17wik"
```
Comment (supakeen) [inline test/e2e/pinnwand.toml:1]: What is this temporary path?

File: test/e2e/testscenarios/test_copy_paste.py [hunk 2]
```diff
@@ -1,27 +1,33 @@
...
-    view_paste_page.click_copy_button(related_paste_number=0)
-    verify_clipboard_contents(first_pasted_text)
+        view_paste_page.click_copy_button(related_paste_number=0)
+        verify_clipboard_contents(first_pasted_text)
 
-    view_paste_page.click_copy_button(related_paste_number=1)
-    verify_clipboard_contents(second_pasted_text)
+        view_paste_page.click_copy_button(related_paste_number=1)
+        verify_clipboard_contents(second_pasted_text)
+    except:
```
Comment (supakeen) [inline test/e2e/testscenarios/test_copy_paste.py:31]: `except Exception:` (it does something different then just `except` :)).

File: test/e2e/pinnwand.toml (hunk 1 above)
Comment (nekhvoya) [reply test/e2e/pinnwand.toml]: removed

File: test/e2e/testscenarios/test_copy_paste.py (hunk 2 above)
Comment (nekhvoya) [reply test/e2e/testscenarios/test_copy_paste.py]: done

**Review by supakeen — APPROVED** [review]

**Pull request merged**